tests/e2e/component.go

```diff
@@ -48,6 +48,14 @@ func componentTests(componentCmdPrefix string) {
 		// Clean up after the test
 		// This is run after every Spec (It)
 		var _ = AfterEach(func() {
+			if CurrentGinkgoTestDescription().Failed {
+				getPod := runCmdShouldPass("oc get pods")
```

Comment:
It would be nice to abstract this whole block of commands to another function, for example, `AferFailed` and call it here.

Reply:
Done